src/app/components/recipe-page/recipe-page.component.ts

```diff
@@ -1,45 +1,46 @@
 import { Component, OnInit } from '@angular/core';
-import { ApiService } from "./../api.service";
+import { ApiService } from './../api.service';
 import { Recipe } from './../models/recipe.model';
 import { ActivatedRoute, Router } from '@angular/router';
-import { switchMap } from 'rxjs/operators';
-
-
-
 
 @Component({
   selector: 'app-recipe-page',
   templateUrl: './recipe-page.component.html',
-  styleUrls: ['./recipe-page.component.css']
+  styleUrls: ['./recipe-page.component.css'],
 })
-
-
-
 export class RecipePageComponent implements OnInit {
   recipe: Recipe;
```

Comment:
Try to define access modificators everywhere. In Angular by default field without anything is public, but that's often not the case for other languages and can be confusing.